packages/main/src/TableHeaderCell.ts

```diff
@@ -45,6 +45,8 @@ class TableHeaderCell extends TableCellBase {
 	 * If the table is in `Popin` mode, the column will move into the popin when
 	 * when the minimum width does not fit anymore.
 	 *
+	 * **Note:** If `minWidth` has value `auto`, the table ensures that the column cannot be smaller than `3rem`.
```

Comment:
I am wondering whether this could be expressed in a positive statement: the column must be wider than

Reply:
Tried something like this: 
> If `minWidth` has the `auto` value, the table ensures that the column is wider than at least `3rem`.